src/schematools/contrib/django/management/commands/create_tables.py

```diff
@@ -41,20 +42,30 @@ def create_tables(
     # First create all models. This allows Django to resolve  model relations.
     models = []
     to_be_skipped = set(skip if skip is not None else [])
+
     for dataset in datasets:
         if not dataset.enable_db or dataset.name in to_be_skipped:
             continue  # in case create_tables() is called by import_schemas
 
         models.extend(schema_models_factory(dataset, base_app_name=base_app_name))
 
+    # Grouping multiple versions of same model by table name
+    models_by_table = defaultdict(list)
+    for model in models:
+        models_by_table[model._meta.db_table].append(model)
+
     # Create all tables
     with connection.schema_editor() as schema_editor:
-        for model in models:
+        for db_table_name, models_group in models_by_table.items():
             # Only create tables if migration is allowed
             # - router allows it (not some external database)
             # - model is managed (not by default)
             # - user overrides this (e.g. developer)
-            db_table_name = model._meta.db_table
+            # - create table for latest version of this dataset group
+            model = list(reversed(sorted(models_group, key=lambda model: model._dataset.version)))[
```

Comment:
Meest recenste versie (hoogste versie nummer) wordt verwerkt als default begrijp ik

Reply:
Only for database table creation, as Minor version should be fully backwards compatible.

Reply:
Untested:
```python
max(models_group, key=lambda model: model._dataset.version)
```
However both this version and your version will probably fail when we have two digit version number for either major, minor or path. Eg:
```python
print(next(reversed(sorted(("1.2.9", "1.1.5", "1.2.10")))))
```
will print `"1.2.9"` whereas it should have been `"1.2.10"`

Reply:
Updated.